src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js

```diff
@@ -59,7 +59,7 @@ function CalCalendarImportController(
   function submit() {
     var notificationMessages = {
       progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
+      success: 'An email will be sent when the import is finished',
```

Comment:
Good improvement but you should add translations for it

```suggestion
      success: 'Calendar is importing, you will receive a reporting email when the import is finished',
```

Reply:
@asmaNode You should make the message clearer:

> The calendar is being imported. You will receive a report email when the import is finished.

I'll give you the Vietnamese translation also:

> Lịch đang được nhập. Bạn sẽ nhận được một thư báo cáo khi quá trình nhập hoàn thành.